### PR TITLE
Fill inputhistory on channel load and more message load

### PR DESCRIPTION
**Pull request by Nachtalb**
as it says in the title. Pretty simple but worth a ton :)

### CODE REVIEW

File: client/js/socket-events/more.js [hunk 1]
```diff
@@ -6,17 +6,24 @@ import socket from "../socket";
 import store from "../store";
 
 socket.on("more", function (data) {
-	const channel = store.getters.findChannel(data.chan);
+	const channel = store.getters.findChannel(data.chan).channel;
```
Comment (xPaw) [inline client/js/socket-events/more.js:9]: Do not do this, findChannel _can_ return null.

Comment (Nachtalb) [reply client/js/socket-events/more.js:9]: Ahh RIP thx, I'll change it

Comment (Nachtalb) [reply client/js/socket-events/more.js:9]: @xPaw would you agree to use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

```suggestion
	const channel = store.getters.findChannel(data.chan)?.channel;
```

Comment (xPaw) [reply client/js/socket-events/more.js:9]: If webpack/babel can deal with it, why not.

Comment (Nachtalb) [reply client/js/socket-events/more.js:9]: Should I lift up the `ecmaVersion` to `2020` or use a comment to tell eslint to ignore it? ==> https://eslint.org/blog/2020/07/eslint-v7.5.0-released#optional-chaining-support

Comment (MaxLeiter) [reply client/js/socket-events/more.js:9]: Yeah, let's bump it to 2020

**Review by MaxLeiter — APPROVED** [review]

**Pull request merged**